lib/plugins/aws/info/getStackInfo.js

```diff
@@ -57,6 +44,24 @@ module.exports = {
       }
 
       return BbPromise.resolve();
+    })
+    .then(() => this.provider.getAccountId())
+    .then((accountId) => {
+      const stage = this.provider.getStage();
+      const region = this.provider.getRegion();
+
+      this.gatheredData.info.functions = [];
+
+      this.serverless.service.getAllFunctions().forEach((func) => {
+        const functionInfo = {};
+        const name = `${this.serverless.service.service}-${stage}-${func}`;
+        const arn = `arn:aws:lambda:${region}:${accountId}:function:${name}`;
```

Comment:
Is it possible to instead get this out of CloudFormation's [ListStackResources](http://docs.aws.amazon.com/AWSCloudFormation/latest/APIReference/API_ListStackResources.html) API as the PhysicalResourceId? For Lambda functions I'm not sure what CFN uses as a physical ID, but it may be the ARN, which would let us avoid auto-building ARNs here.